src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1212,6 +1212,18 @@ private AdsDataProvider() {
     }
 
   @DataProvider
+  public static Object[][] adsVideoFrequencyCapping() {
+    return new Object[][]{
+        {
+            new Page(WIKI_SPECIAL, "SyntheticTests/RTB/Prebid.js/Veles/Incontent" + "?InstantGlobals.wgAdDriverOutstreamVideoFrequencyCapping=[1/2pv]")
```

Comment:
I'm surprised `?InstantGlobals.wgAdDriverOutstreamVideoFrequencyCapping=[1/2pv]` works :japanese_ogre:  But it indeed does :+1: